dateconv/dateconv.go

```diff
@@ -134,6 +134,32 @@ func TotalDaysInBSYear(year int) (int, error) {
 	return sum, nil
 }
 
+// now returns the current time
+var now = func() time.Time {
+	return time.Now()
+}
+
+// TotalDaysSpanned returns the total number of days spanned in the
+// current year inclusive of the current day.
+func TotalDaysSpanned() (int, error) {
+	bsDate := ToBS(now())
+	days, ok := bsDaysInMonthsByYear[bsDate.year]
+
```

Comment:
Line 146 should be blank instead. In general grouping the `if !ok` or `if err != nil` handlers with their respective initializations makes sense to me.
```go
bsDate := ..

days, ok := ..
if !ok  {
}

Reply:
Ok chief. 👍